R/topics.R

```diff
@@ -92,6 +92,17 @@ match_env <- function(topics) {
         purrr::map_lgl(any)
 
       which(match & is_public(internal))
+    },
+    # Implementation here: cannot exploit has_concept() above,
```

Comment:
Can you please remove this comment and add a NEWS bullet? Then I can merge

Reply:
Done in mikldk/pkgdown@85c7cc801032e65710ef8e9f906db328f444cdfb.